tools/cli/admin.go

```diff
@@ -245,6 +245,40 @@ func newAdminDomainCommands() []cli.Command {
 				AdminGetDomainIDOrName(c)
 			},
 		},
+		{
+			Name:  "migrate",
```

Comment:
Actually we don't have to merge this to master. This command is only for temporary and we won't use it anymore after migration. Adding a new command here will just confuse our customer. I would suggest making a private branch. And you don't have to wait for release and then adhoc deployment script to publish to every adhoc host.